src/components/TutorialsTop.js

```diff
@@ -32,6 +32,11 @@ export function TutorialsTop({ highlightId } = {}) {
                 </video>
                 <div className="text-sm">Ink Detection</div>
             </a>
```

Comment:
```suggestion
            </a>
            <div className="hidden sm:flex mx-2 mb-2 flex-col items-center">
                <div className="relative leading-[150px] py-4 w-[16px] text-center">→</div>
                <div className="text-sm">&nbsp;</div>
            </div>
```
Adds column styling based on existing content.